### ISSUE TITLE
Bug: numeral is not in sync with objects

### ISSUE THREAD

**Issue by amanda-phet** [phetsims/number-play#39]
This bug came up in a student interview. We'd like QA to find the steps to reproduce so that @chrisklus can tackle it. Originally found on iPad (not sure of exact OS or model, but it's brand new, so should be up to date). 

Video of bug: https://drive.google.com/file/d/1UVavAf22wnKStI25kdysiZkrSRASzmDz/view?usp=sharing

**Comment by Nancy-Salpepi** [comment]
@amanda 
I was able to replicate only once using the iPad by dragging the apple over the bowl containing the additional apples. The number at the top of the screen changed from 12 to 11 (as it did 17sec into the video) but I am not able to nail down exact steps at this point.

**Comment by KatieWoe** [comment]
I think I got the cause. You need to drop the apple/object and pick it up again before it goes in the bowl. To the point where it is basically a finger twitch. I can reproduce this consistently doing this. This seems to create a bugged apple that doesn't show up in the number count and behaves oddly in the play area.

https://user-images.githubusercontent.com/41024075/129810354-f862f8a8-0148-4dac-b677-073539e4c06e.MP4

**Comment by KatieWoe** [comment]
Linking to dev test https://github.com/phetsims/qa/issues/696

**Comment by KatieWoe** [comment]
This can happen on other screens with other objects in other scenes as well.
![992B5AB0-77D7-49B6-B75C-AB0D69C06713](https://user-images.githubusercontent.com/41024075/129941992-74f8c843-efec-47b2-816a-d6c8f35504db.png)

**Comment by amanda-phet** [comment]
Thanks @KatieWoe for figuring this out!

**Comment by chrisklus** [comment]
I committed a temporary fix above, which I will open a new issue for assuming this is working correctly as is. @KatieWoe could confirm this is no longer reproducible on master?

**Comment by KatieWoe** [comment]
I saw a crash/freeze happen on master when trying to reproduce this. It doesn't always happen, and seems to happen if I miss trying to catch the object when recreating the bug.
<img width="959" alt="mayfixnewbug" src="https://user-images.githubusercontent.com/41024075/130690566-5b105f91-3894-490f-a492-627f9e2767bf.png">

**Comment by chrisklus** [comment]
Thanks @KatieWoe! That was very helpful for me to figure out the actual source of this issue - I've been unable to figure out why setting the paper numbers to `pickable: false` was blocking mouse inputs but not touch inputs. I thought my temporary fix of just removing that input listener was working (but as you found out, it was just crashing the sim - i must have only tested once per sim run on my iPhone and not realized it was crashing). Turns out, `pickable: false` was correctly blocking mouse and touch inputs, but there is a *separate* input listener that was added to the background of the play area of type ClosestDragListener, whose purpose was forwarding *only* touch inputs to the nearest paper number within 30 pixels of the drag event (in order to dilate it's touch areas without expanding its bounds is my guess). Once it gets a drag event for a closest paper number, it forwards the drag to the drag listener of the paper number itself, so the crash from above was caused by that drag listener being deleted from my attempted fix earlier.

I removed my first commit in this issue and blocked the ClosestDragListener drags from forwarding to the paper number's drag listener instead, and that seems to be working nicely. @KatieWoe could you please take a look again and close if all looks good? Thanks again for all of your help on this.

**Comment by chrisklus** [comment]
I forgot to mention that this change involved `counting-common`, so i think that was either not pulled or still cached.
